### PR TITLE
Feature/impl-1340

### PR DESCRIPTION
**Pull request by fedecl-vv**
Build Automation: Submit appeal with invalid denial code from Public link

### CODE REVIEW

File: Keywords/pages/Page_Licensure_Unit_Appeals.groovy [hunk 1]
```diff
@@ -51,4 +52,15 @@ public class Page_Licensure_Unit_Appeals {
 
 		WebUI.verifyElementAttributeValue(inputEmail, 'value', email, 0)
 	}
+	
+	@Keyword
+	def verifyErrorAlert (String alertObj, String errorText) {
+		if (alertObj.contains(errorText)) {
+			// Close Browser
+			WebUI.closeBrowser()
```
Comment (patriciorodriguezOT) [inline Keywords/pages/Page_Licensure_Unit_Appeals.groovy]: @fedecl-ot Let's put the step CloseBrowser on the Test Case, not on the Keyword

File: Keywords/pages/Page_Licensure_Unit_Appeals.groovy [hunk 2]
```diff
@@ -51,4 +52,15 @@ public class Page_Licensure_Unit_Appeals {
...
+		} else {
+			// Stop tc execution
+			KeywordUtil.markErrorAndStop('\nThe test case can not be completed.')
```
Comment (patriciorodriguezOT) [inline Keywords/pages/Page_Licensure_Unit_Appeals.groovy]: 1 - Use markFailed, markErrorAndStop are used when TCs can not be executed for some reason.

2 - Please, add some more information why the Test Case has failed

File: Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code.tc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestCaseEntity>
+   <description>On this Test Case the Admin User approve the License Application.</description>
+   <name>Appeal Denied License Application - Invalid denial code</name>
```
Comment (patriciorodriguezOT) [inline Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code.tc]: 1- The name of the Test Case should be "Submit appeal with invalid denial code from Public link"

2 - Create a folder Appeal and put the test cases related to that topic there.

File: Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+   <pageLoadTimeout>30</pageLoadTimeout>
+   <pageLoadTimeoutDefault>true</pageLoadTimeoutDefault>
+   <rerunFailedTestCasesOnly>false</rerunFailedTestCasesOnly>
+   <rerunImmediately>true</rerunImmediately>
+   <testSuiteGuid>32164761-29a0-4405-add3-11d985eab8b5</testSuiteGuid>
+   <testCaseLink>
+      <guid>1d9c2ddd-2cc3-4f63-bf25-884bde29c512</guid>
+      <isReuseDriver>false</isReuseDriver>
+      <isRun>true</isRun>
+      <testCaseId>Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code</testCaseId>
```
Comment (patriciorodriguezOT) [inline Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts]: Please, move the Test Suite to Individual Test Cases/Appeal (Create the folder if it is necessary). The name of the Test Suite should be the same of the Test Case.

**Review by patriciorodriguezOT — CHANGES_REQUESTED** [review]
Please, review comments on all files.

File: Keywords/pages/Page_Licensure_Unit_Appeals.groovy (hunk 1 above)
Comment (fedecl-ot) [reply Keywords/pages/Page_Licensure_Unit_Appeals.groovy]: Done

File: Keywords/pages/Page_Licensure_Unit_Appeals.groovy (hunk 2 above)
Comment (fedecl-ot) [reply Keywords/pages/Page_Licensure_Unit_Appeals.groovy]: Done

File: Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code.tc (hunk 3 above)
Comment (fedecl-ot) [reply Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code.tc]: Done

File: Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts (hunk 4 above)
Comment (fedecl-ot) [reply Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts]: Done

**Comment by fedecl-vv** [comment]
All changes requested were implemented to approve PR.

File: Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts [hunk 5]
```diff
@@ -15,6 +15,6 @@
       <guid>1d9c2ddd-2cc3-4f63-bf25-884bde29c512</guid>
       <isReuseDriver>false</isReuseDriver>
       <isRun>true</isRun>
-      <testCaseId>Test Cases/Licensure Unit License Application/Submit appeal with invalid denial code from Public link</testCaseId>
+      <testCaseId>Test Cases/Licensure Unit License Application/Appeal/Submit appeal with invalid denial code from Public link</testCaseId>
```
Comment (patriciorodriguezOT) [inline Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts]: @fedecl-ot The "Appeal" folder should be on the Test Case folder, not inside the "Licensure Unit License Application"

File: Test Suites/Individual Test Cases/License Application/Appeal/Submit appeal with invalid denial code from Public link.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <TestSuiteEntity>
    <description>Submit appeal with invalid denial code from Public link</description>
-   <name>Appeal Denied License Application - Invalid denial code</name>
+   <name>Submit appeal with invalid denial code from Public link</name>
```
Comment (patriciorodriguezOT) [inline Test Suites/Individual Test Cases/License Application/Appeal/Submit appeal with invalid denial code from Public link.ts]: @fedecl-ot please move the Test Suite to the folder "Test Suites/Individual Test Cases/Appeal/". The folder Appeal should be on the "Individual Test Case" folder, not inside of "License Application"

**Comment by patriciorodriguezOT** [comment]
@fedecl-ot please view my comments regarding the Appeal folder.

File: Test Suites/Individual Test Cases/License Application/Appeal/Submit appeal with invalid denial code from Public link.ts (hunk 6 above)
Comment (fedecl-ot) [reply Test Suites/Individual Test Cases/License Application/Appeal/Submit appeal with invalid denial code from Public link.ts]: Done

File: Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts (hunk 5 above)
Comment (fedecl-ot) [reply Test Suites/Individual Test Cases/License Application/Appeal Denied License Application - Invalid denial code.ts]: Done

**Comment by fedecl-vv** [comment]
All changes requested were implemented to approve PR.

**Review by patriciorodriguezOT — APPROVED** [review]
The Feature/impl-1340 is approved to merge into develop.

**Pull request merged**